pep_sphinx_extensions/pep_theme/static/style.css

```diff
@@ -226,14 +231,17 @@ div.table-wrapper {
 table {
     width: 100%;
     border-collapse: collapse;
-    border-top: 1px solid var(--colour-background-accent-light);
-    border-bottom: 1px solid var(--colour-background-accent-light);
+    border: 1px solid var(--colour-background-accent-strong);
 }
 table caption {
     margin: 1rem 0 .75rem;
 }
-table tbody tr:nth-of-type(odd) {
-    background-color: var(--colour-background-accent-light);
+table thead tr {
+    background-color: var(--colour-admonition);
```

Comment:
This is a bit awkward. Maybe declare the variable as...

```
    --colour-background-accent-medium: var(--light, #ddd) var(--dark, #333);
```

And use it for admonition and table headers?

Reply:
Yeah, that makes a lot more sense, thanks—initially, both of these used `--color-background-accent-strong`, but then I switched the admonition to a lighter color to improve contrast/legibility since I made the strong color darker for the table borders, and gave it its own color name like the rest of the admonition types. However, then after I removed the zebra striping, I was able to make the table header lighter and ended up using the same color, but neglected to change the name. I went ahead and implemented it as you suggested, thanks :+1: